### PR TITLE
fix for draggable widgets in RTL + added ignore function for datepicker in popup in dialog lib

### PR DESCRIPTION
**Pull request by udidol**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Comment by KingYes** [comment]
@kobizz @udidol Please change the version in embed function.

**Comment by udidol** [comment]
> @kobizz @udidol Please change the version in embed function.

Done

File: includes/elements/section.php [hunk 1]
```diff
@@ -418,16 +418,26 @@ protected function _register_controls() {
...
 				'default' => '',
 				'options' => [
 					'' => __( 'Default', 'elementor' ),
 					'top' => __( 'Top', 'elementor' ),
-					'middle' => __( 'Middle', 'elementor' ),
+					'center' => __( 'Middle', 'elementor' ),
 					'bottom' => __( 'Bottom', 'elementor' ),
-				],
-				'prefix_class' => 'elementor-section-content-',
+                    'space-between' => __( 'Space Between', 'elementor' ),
```
Comment (kobizz) [inline includes/elements/section.php]: Use tabs instead of spaces

File: includes/elements/section.php [hunk 2]
```diff
@@ -418,16 +418,26 @@ protected function _register_controls() {
...
+                    'space-around' => __( 'Space Around', 'elementor' ),
+                    'space-evenly' => __( 'Space Evenly', 'elementor' ),
+				],
+                'selectors_dictionary' => [
+                    'top' => 'flex-start',
+                    'bottom' => 'flex-end',
+                ],
+                'selectors' => [
+                    '{{WRAPPER}} > .elementor-container > .elementor-row > .elementor-column > .elementor-column-wrap' => 'align-items: {{VALUE}}',
+                    '{{WRAPPER}} > .elementor-container > .elementor-row > .elementor-column > .elementor-column-wrap > .elementor-widget-wrap' => 'align-content: {{VALUE}}',
```
Comment (kobizz) [inline includes/elements/section.php]: Remove this line

File: includes/elements/section.php [hunk 3]
```diff
@@ -418,16 +418,26 @@ protected function _register_controls() {
 		$this->add_control(
 			'content_position',
 			[
-				'label' => __( 'Content Position', 'elementor' ),
+				'label' => __( 'Vertical Align', 'elementor' ),
```
Comment (kobizz) [inline includes/elements/section.php]: Why?

File: includes/elements/section.php [hunk 4]
```diff
@@ -418,16 +418,26 @@ protected function _register_controls() {
...
 				'type' => Controls_Manager::SELECT,
 				'default' => '',
 				'options' => [
 					'' => __( 'Default', 'elementor' ),
 					'top' => __( 'Top', 'elementor' ),
-					'middle' => __( 'Middle', 'elementor' ),
+					'center' => __( 'Middle', 'elementor' ),
```
Comment (kobizz) [inline includes/elements/section.php]: Don't change this value. Add it to the `selectors_dictionary` instead

**Review by kobizz — CHANGES_REQUESTED** [review]

File: includes/elements/section.php (hunk 2 above)
Comment (udidol) [reply includes/elements/section.php]: Vertical positioning does not work properly without this line. 
`flex-start`, `center`, and `flex-end` are `align-item` values, while `space-between`, `space-around` and `space-evenly` are `align-content` values. 
That is why these settings are divided to two separate selectors (also in `column.php`).

File: includes/elements/section.php (hunk 3 above)
Comment (udidol) [reply includes/elements/section.php]: Shilo asked for it. 
In column settings, the same control is labeled Vertical Align, I assume he wanted uniformity.
I think this change was agreed upon a while ago and no one got around to doing it.

**Review by kobizz — APPROVED** [review]

**Pull request merged**